modules/text/samples/webcam_demo.cpp

```diff
@@ -140,120 +183,79 @@ int main(int argc, char* argv[])
     vector< Ptr<OCRHMMDecoder> > decoders;
     for (int o=0; o<num_ocrs; o++)
     {
-      decoders.push_back(OCRHMMDecoder::create(loadOCRHMMClassifierNM("OCRHMM_knn_model_data.xml.gz"),
-                                               voc, transition_p, emission_p));
+        decoders.push_back(OCRHMMDecoder::create(loadOCRHMMClassifierNM("OCRHMM_knn_model_data.xml.gz"),
+                           voc, transition_p, emission_p));
     }
     cout << " Done!" << endl;
 
-    //cout << "TIME_OCR_INITIALIZATION_ALT = "<< ((double)getTickCount() - t_r)*1000/getTickFrequency() << endl;
-
-
-    int cam_idx = 0;
-    if (argc > 1)
-        cam_idx = atoi(argv[1]);
-
-    VideoCapture cap(cam_idx);
-    if(!cap.isOpened())
-    {
-        cout << "ERROR: Cannot open default camera (0)." << endl;
-        return -1;
-    }
-
-    while (cap.read(frame))
+    while ( true )
     {
         double t_all = (double)getTickCount();
 
         if (downsize)
             resize(frame,frame,Size(320,240));
 
         /*Text Detection*/
-
-        cvtColor(frame,grey,COLOR_RGB2GRAY);
-        grey.copyTo(orig_grey);
+        cvtColor(frame,gray,COLOR_BGR2GRAY);
+        gray.copyTo(orig_gray);
```

Comment:
It looks like `orig_gray` isn't used anywhere else.

Reply:
@terfendail  thanks for review. `orig_gray `removed.